pkg/option/flags.go

```diff
@@ -295,4 +343,5 @@ func AddFlags(flags *pflag.FlagSet) {
 
 	flags.Bool(KeyGenerateDocs, false, "Generate documentation in YAML format to stdout")
 
+	flags.String(KeyCgroupRate, "", "Base sensor events cgroup rate (events,interval)")
```

Comment:
I guess empty ("") means that no cgroup rate is applied? If so, let's document this in the option.

Reply:
yes, will add the doc